5-Heroku/server/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const toDoRouter = require("./routes/todos");
+const cors = require("cors");
+const mongoose = require("mongoose");
+require("dotenv").config('5-Heroku/server/.env');
+
+mongoose.connect(
+  process.env.MONGODB_URL,
+  {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  }
+);
+const db = mongoose.connection;
+db.once("open", () => {
+  console.log("connection started");
+});
+const app = express();
+
+app.use(cors({ credentials: true, origin: "http://localhost:8080" }));
```

Comment:
you're allowing `localhost:8080` to communicate with your server. but what will happen in production? the url may vary so i'd expect this to be set dynamically. it is extremely important not to have dev/test code merged